packages/assets-controllers/src/TokenListController.test.ts

```diff
@@ -1294,7 +1294,7 @@ describe('TokenListController', () => {
     });
   });
 
-  describe('executePoll', () => {
+  describe('_executePoll', () => {
```

Comment:
Do we want to keep these tests if we are essentially making this method private? [We don't conventionally test private code.](https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#dont-directly-test-private-code) Is there some other place we could move these tests to?

Reply:
removed this block and moved the test into the `startPollingByNetworkClientId` test block: https://github.com/MetaMask/core/pull/1810/commits/7a404128b07d3eb446aae9a8c84004eadca3e10c